README.md

```diff
@@ -13,33 +13,24 @@ between both the server and the client) for SEO reasons.
 [![Dependency Status](https://david-dm.org/quran/quran.com-frontend.svg)](https://david-dm.org/quran/quran.com-frontend) [![devDependency Status](https://david-dm.org/quran/quran.com-frontend/dev-status.svg)](https://david-dm.org/quran/quran.com-frontend#info=devDependencies)
 [![Code Climate](https://codeclimate.com/github/quran/quran.com-frontend.png)](https://codeclimate.com/github/quran/quran.com-frontend)
 
-## Getting started
-Simply clone this repo, then run `npm install` to install all the required node_modules.
-From there, you are ready to go!
+## How to contribute
+We trust that
+you will not steal this, this is at the end of the day for the sake of Allah and we
```

Comment:
I would suggest adding a license to the project, and I would suggest the GPL license. I wouldn't worry about people copying parts of the project, or even forking it and making it better, as long as their intentions are good, and God knows the intentions of every body.